### PR TITLE
HW-Adventure-Habtamu Abele

### PR DESCRIPTION
**Pull request by HabRonan**
comfort-4.5
completeness- 4.5

### CODE REVIEW

File: starter/script.js [hunk 1]
```diff
@@ -1 +1,92 @@
...
+                          alert("your body can't survive this")
+                        break;
+                   case ("saturn"):
+                          alert("your body can't survive this")
+                          break;
+                   case ("neptune"):
+                        alert("your body can't survive this")
+                          break;
+
+                         }
```
Comment (amaseda) [inline starter/script.js:89]: What happens if this switch statement is triggered but the user input does not match any of the planets you have specified?

File: starter/script.js [hunk 2]
```diff
@@ -1 +1,92 @@
...
+        alert("you can't go back to the past unless you have a time machine!")
+      }
+
+}
+
+if (planet == "pandora"){
+        var type = prompt("great Choice, what kind would you like to be? a human, an avatar or a na'vi")
+        var check = ["human", "Avatar", "na'vi" ];
+        var text = " ";
+        var i;
```
Comment (amaseda) [inline starter/script.js:51]: With for loops you do not have to define `i` prior to the loop (unless you have a specific reason for doing so). Instead, this is a common practice...


```js
for(var i=0; i<arr.length; i++){
  // code goes here
}
```
> Note that we are initializing `i` inside `for`

**Comment by amaseda** [comment]
:+1:

File: starter/script.js (hunk 1 above)
Comment (HabRonan) [reply starter/script.js:89]: ya, I had a default statement but that one gets executed all the time and then I took it out.
I guess I have to look into that.

File: starter/script.js (hunk 2 above)
Comment (HabRonan) [reply starter/script.js:51]: Alright, Thanks.

**Pull request closed without merging**